crates/turbopack-tests/tests/snapshot/runtime/default_dev_runtime/output/79fb1_turbopack-tests_tests_snapshot_runtime_default_dev_runtime_input_index_e60ecd.js

```diff
@@ -1,1474 +1,8 @@
-            };
-            chunkResolvers1.set(chunkPath1, resolver1);
-        }
-        return resolver1;
-    }
-    function deleteResolver1(chunkPath1) {
-        chunkResolvers1.delete(chunkPath1);
-    }
-    /**
-   * Loads the given chunk, and returns a promise that resolves once the chunk
-   * has been loaded.
-   */ async function doLoadChunk1(chunkPath1, source1) {
-        const resolver1 = getOrCreateResolver1(chunkPath1);
-        if (resolver1.resolved) {
-            return resolver1.promise;
-        }
-        if (source1.type === SourceType.Runtime) {
-            // We don't need to load chunks references from runtime code, as they're already
-            // present in the DOM.
-            if (chunkPath1.endsWith(".css")) {
-                // CSS chunks do not register themselves, and as such must be marked as
-                // loaded instantly.
-                resolver1.resolve();
-            }
-            // We need to wait for JS chunks to register themselves within `registerChunk`
-            // before we can start instantiating runtime modules, hence the absence of
-            // `resolver.resolve()` in this branch.
-            return resolver1.promise;
-        }
-        const chunkUrl1 = `/${getChunkRelativeUrl(chunkPath1)}`;
-        if (chunkPath1.endsWith(".css")) {
-            const link1 = document.createElement("link");
-            link1.rel = "stylesheet";
-            link1.href = chunkUrl1;
-            link1.onerror = ()=>{
-                resolver1.reject();
-            };
-            link1.onload = ()=>{
-                // CSS chunks do not register themselves, and as such must be marked as
-                // loaded instantly.
-                resolver1.resolve();
-            };
-            document.body.appendChild(link1);
-        } else if (chunkPath1.endsWith(".js")) {
-            const script1 = document.createElement("script");
-            script1.src = chunkUrl1;
-            // We'll only mark the chunk as loaded once the script has been executed,
-            // which happens in `registerChunk`. Hence the absence of `resolve()` in
-            // this branch.
-            script1.onerror = ()=>{
-                resolver1.reject();
-            };
-            document.body.appendChild(script1);
-        } else {
-            throw new Error(`can't infer type of chunk from path ${chunkPath1}`);
-        }
-        return resolver1.promise;
-    }
-})();
-function _eval({ code, url, map }) {
-    code += `\n\n//# sourceURL=${location.origin}/${CHUNK_BASE_PATH}${url}`;
-    if (map) code += `\n//# sourceMappingURL=${location.origin}/${CHUNK_BASE_PATH}${map}`;
-    return eval(code);
-}
-const chunksToRegister = globalThis.TURBOPACK;
-globalThis.TURBOPACK = { push: registerChunk };
-chunksToRegister.forEach(registerChunk);
-})();
-
-
-//# sourceMappingURL=79fb1_turbopack-tests_tests_snapshot_runtime_default_dev_runtime_input_index_e60ecd.js.map
\ No newline at end of file
+{
```

Comment:
@sokra Seems like this file is a source map. Any idea how can this happen?

Reply:
Yeah, I've seen this bug. Doesn't seem to be related to SWC update.